pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -273,6 +270,29 @@ public static void main(String[] args) throws Exception {
         log.info("Cluster metadata for '{}' setup correctly", arguments.cluster);
     }
 
+    private static ZooKeeper initZk(String connection, int sessionTimeout) throws Exception {
```

Comment:
@murong00 can you rewrite unit tests for this method?

Reply:
Added.